docs/kb/kb0004-akv2aks.md

```diff
@@ -132,20 +132,47 @@ Useful links:
         userAssignedIdentityID: $kvManagedIdentity # This is the guid for the kv managed identity
         keyvaultName: $kv.name # This is your key vault name
         tenantId: $tenantId # This is the guid of your tenant
-        secretRanges:
-        - secretName: cluedin-login-details # This is how the secret will appear within Kubernetes Secrets once synchronised
-          secretType: Opaque # For most secrets, leave as Opaque
-          secretKeys:
-            password: cluedin-login-password # The left side (Key) is the name in the Kubernetes secret object. The right side (Value) is the AKV reference which will grab its value.
-            username: cluedin-login-username
+        secretProviderClasses:
+          cluedin-server: # For every key under `secretProviderClasses` a new secret provider class will be created with the same name an `-sync` appended.
+          - secretName: cluedin-login-details # This is how the secret will appear within Kubernetes Secrets once synchronised
+            secretType: # [OPTIONAL] Defaults to Opaque. But you can specify any supported secretType
+            secretKeys:
+              password: cluedin-login-password # The left side (Key) is the name in the Kubernetes secret object. The right side (Value) is the AKV reference which will grab its value.
+              username: cluedin-login-username
 
     infrastructure:
       cert-manager:
         #enabled: false # Only set to false if using an uploaded frontend certificate.
     ```
-    **NOTE**: If you are using the certificate upload as part of your setup, you **must** disable cert-manager by setting `enabled: false`. You also must set `frontendCrt` to a value. It will be mounted as `cluedin-frontend-crt`. This secret name cannot change.
+    **NOTE**: If you are using the certificate upload as part of your setup, you **must** disable cert-manager by setting `enabled: false`. You also must set `frontendCrt` to a value. It will be mounted as `cluedin-frontend-crt` on `cluedin-server`. This secret name cannot change.
```

Comment:
```suggestion
    {:.important}
    If you are using the certificate upload as part of your setup, you **must** disable cert-manager by setting `enabled: false`. You also must set `frontendCrt` to a value. It will be mounted as `cluedin-frontend-crt` on `cluedin-server`. This secret name cannot change.
```